deploy/operator.yaml

```diff
@@ -33,6 +33,75 @@ roleRef:
   name: config-map-reader
   apiGroup: rbac.authorization.k8s.io
 ---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: seccomp-operator-profile
+  namespace: seccomp-operator
+data:
+  seccomp-operator-v0.1.json: |-
```

Comment:
What was the reason the version this? Do you think we can omit the version string here?

Reply:
The idea behind this was to avoid overwriting profiles when deploying. On such instances a rollback would still have side-effects (specially if the new profile removed allowed syscalls).

I think we can remove that version for the time being, and do something around those lines when a new profile is not backwards compatible. What do you reckon?

Reply:
I agree. OTOH could versionize this profile on its own, so a v1 should a should probably also work. I’m not opinionated any more and leave it up to you.